build.gradle

```diff
@@ -40,6 +40,7 @@ configurations {
 }
 
 dependencies {
+  compile group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '1.6.0'
```

Comment:
Hi! Sorry for delay. It's an minor issue.
It looks like for gradle it is hard to check it.
Let's skip it now. I will have look at this later.